tests/unit/s2n_client_psk_extension_test.c

```diff
@@ -273,6 +328,38 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_free(&out));
         }
 
+        /* Send a resumption psk identity */
+        {
+            struct s2n_stuffer out = { 0 };
+            EXPECT_SUCCESS(s2n_stuffer_growable_alloc(&out, 0));
+
+            struct s2n_connection *conn;
+            EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_CLIENT));
+
+            struct s2n_psk *psk = NULL;
+            EXPECT_OK(s2n_array_pushback(&conn->psk_params.psk_list, (void**) &psk));
+            EXPECT_OK(s2n_psk_init(psk, S2N_PSK_TYPE_RESUMPTION));
+            EXPECT_SUCCESS(s2n_psk_set_identity(psk, test_identity, sizeof(test_identity)));
+            psk->hmac_alg = S2N_HMAC_SHA384;
+            psk->ticket_age_add = 10;
+
+            uint64_t issue_time = 0;
+            POSIX_GUARD(conn->config->wall_clock(conn->config->sys_clock_ctx, &issue_time));
+            psk->ticket_issue_time = issue_time;
+
+            EXPECT_SUCCESS(s2n_client_psk_extension.send(conn, &out));
+
+            EXPECT_SUCCESS(s2n_stuffer_skip_read(&out, sizeof(uint16_t) /* identity_list_size */ +
+                sizeof(uint16_t) /* identity_size */ + sizeof(test_identity)));
+
+            uint32_t obfuscated_ticket_age = 0;
+            EXPECT_SUCCESS(s2n_stuffer_read_uint32(&out, &obfuscated_ticket_age));
+            EXPECT_TRUE(obfuscated_ticket_age > 0);
+
```

Comment:
Should this be the end of the stuffer? Can you say anything about what should be left? Because you're not checking the exact obfuscated_ticket_age, it makes me a little nervous that you could be reading the wrong data.

Reply:
I can freeze the wall clock and calculate exactly what obfuscated_ticket_age should be, if you think that would be more comprehensive. Just wasn't sure if it was needed here.